src/CardSchemeComponent.res

```diff
@@ -0,0 +1,37 @@
+module CoBadgeCardSchemeDropDown = {
```

Comment:
We have our drop down component cam we use that only?

Reply:
This drop down requires some additional customisation like the label (select a card brand) should be shown in the dropdown list and not as a hint in select, only the dropdown arrow (chevron) should be visible while selecting. Exposing additional props to achieve things like these could have made our reusable component messy. As these requirements are unique to this co badge drop down. Now this card scheme component can be reused anywhere in the SDK wherever the user is required to select the card scheme.